egs/wsj/s5/steps/libs/nnet3/xconfig/utils.py

```diff
@@ -14,14 +14,16 @@
 # Given a list of objects of type XconfigLayerBase ('all_layers'),
 # including at least the layers preceding 'current_layer' (and maybe
 # more layers), return the names of layers preceding 'current_layer'
+# other than layers of type 'auxiliary'.
```

Comment:
change to:
`other than layers of type 'auxiliary', which correspond to component-node names
 from an existing model that we are adding layers to, and which can't be used in
 expressions like [-1].`
obviously make sure line length is OK.